kedro/framework/cli/__init__.py

```diff
@@ -1,7 +1,11 @@
 """``kedro.framework.cli`` implements commands available from Kedro's CLI.
 """
 
-from .cli import main
-from .utils import command_with_verbosity, load_entry_points
+# The constant need to be defined first otherwise it causes circular depdencies
+ORANGE = (255, 175, 0)
+BRIGHT_BLACK = (128, 128, 128)
```

Comment:
Can these go in `kedro.utils`? So you wouldn't have to define them first.

Reply:
It can, but I think `cli` is the correct scope as this only affect CLI color. Alternative is keep it in `cli/cli.py` because this is the only file that get used at the moment.